From 768d2909053e74da35323f3ec6bce1226a03dee8 Mon Sep 17 00:00:00 2001
From: YumeMichi <do4suki@gmail.com>
Date: Wed, 21 Feb 2018 23:09:27 +0800
Subject: [PATCH 3/4] Add mkshrc with some aliases

this will simple overrule the one from external/mkshrc
not the nicest solution but this should not be rocket science

Change-Id: I449e313d55aefda8b23cfb87ccff580e00eeaa19
---
 config/common.mk           |  4 ++++
 prebuilt/common/etc/mkshrc | 27 +++++++++++++++++++++++++++
 2 files changed, 31 insertions(+)
 create mode 100644 prebuilt/common/etc/mkshrc

diff --git a/config/common.mk b/config/common.mk
index 63b42df..01d1ce2 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -130,6 +130,10 @@ PRODUCT_PACKAGES += \
 PRODUCT_COPY_FILES += \
     vendor/lineage/prebuilt/common/install/gapps-config.txt:install/gapps-config.txt
 
+# mkshrc
+PRODUCT_COPY_FILES += \
+    vendor/lineage/prebuilt/common/etc/mkshrc:system/etc/mkshrc
+
 # Custom Lineage packages
 PRODUCT_PACKAGES += \
     AudioFX \
diff --git a/prebuilt/common/etc/mkshrc b/prebuilt/common/etc/mkshrc
new file mode 100644
index 0000000..0e57d57
--- /dev/null
+++ b/prebuilt/common/etc/mkshrc
@@ -0,0 +1,27 @@
+# Copyright (c) 2010, 2012, 2013, 2014
+#	Thorsten Glaser <tg@mirbsd.org>
+# This file is provided under the same terms as mksh.
+#-
+# Minimal /system/etc/mkshrc for Android
+#
+# Support: https://launchpad.net/mksh
+
+: ${HOSTNAME:=$(getprop ro.product.device)}
+: ${HOSTNAME:=android}
+: ${TMPDIR:=/data/local/tmp}
+export HOSTNAME TMPDIR
+
+if (( USER_ID )); then PS1='$'; else PS1='#'; fi
+PS4='[$EPOCHREALTIME] '; PS1='${|
+	local e=$?
+
+	(( e )) && REPLY+="$e|"
+
+	return $e
+}$HOSTNAME:${PWD:-?} '"$PS1 "
+
+alias ls='ls --color=auto'
+alias grep='grep --color=always'
+alias rgrep='grep -R'
+alias ll='ls -al'
+alias lz='ls -alZ'
-- 
2.7.4

